src/LinearSolverTypes_Multigrid.f90

```diff
@@ -728,8 +756,13 @@ SUBROUTINE setSmoother_LinearSolverType_Multigrid(solver,smoother,iLevel,num_smo
       CALL KSPSetType(ksp_temp,KSPPREONLY,iperr)
       CALL KSPGetPC(ksp_temp,pc_temp,iperr)
       CALL PCSetType(pc_temp,PCLU,iperr)
-      IF(solver%MPIparallelEnv%nproc > 1) &
-          CALL PCFactorSetMatSolverPackage(pc_temp,MATSOLVERSUPERLU_DIST,iperr)
+      IF(solver%MPIparallelEnv%nproc > 1) THEN
+#if ((PETSC_VERSION_MAJOR>=3) && (PETSC_VERSION_MINOR>6))
+        CALL PCFactorSetMatSolverType(pc_temp,MATSOLVERSUPERLU_DIST,iperr)
+#else
+        CALL PCFactorSetMatSolverPackage(pc_temp,MATSOLVERSUPERLU_DIST,iperr)
+#endif
```

Comment:
there are probably other issues with the multigrid stuff (I disabled for 3.12) that it's possible I didn't change it because the unit test didn't fail with it...the other changes here were necessary for compilation

Reply:
Alright.  Did I miss where you disabled it?  Or was it in MPACT somewhere?

Reply:
testLinearSolvers_Multigrid...there's a version check around the subtests

Reply:
whoops, I totally missed that there wasn't' a corresponding `#else` for that one.